GitCommands/PathUtil.cs

```diff
@@ -16,6 +16,11 @@ public static partial class PathUtil
         public static readonly char PosixDirectorySeparatorChar = '/';
         public static readonly char NativeDirectorySeparatorChar = Path.DirectorySeparatorChar;
 
+        /// <summary>The user's profile folder path.</summary>
+        // TODO verify whether the user profile contains forwards/backwards slashes on other platforms
```

Comment:
Is this TODO for now or for later?

Reply:
It was already there and I just moved it and I didn't know exactly what to do with it...
Maybe someone using WSL could have a look but if no one ever complained, I think it could be removed...

Reply:
All good then, merge at will.

Thank you for your efforts.